packages/console/src/pages/CustomizeJwt/CustomizerItem/index.tsx

```diff
@@ -36,7 +36,7 @@ function CustomizerItem({ tokenType }: Props) {
 
     if (confirm) {
       await api.delete(apiLink);
-      await mutate(apiLink);
+      await mutate(apiLink, undefined);
```

Comment:
fix a bug. 
After the deleting request, and calling the SWR to mutate, the get request will throw a 404 error which is expected. 
However, SWR won't update the data value to undefined due to a request exception occurred. We need to manually set the value to undefined.

Reply:
nit: can add the explaination to code comments.